ai_edge_torch/generative/examples/gemma/convert_to_tflite.py

```diff
@@ -45,9 +45,9 @@ def convert_gemma_to_tflite(
       checkpoint_path, kv_cache_max_len=kv_cache_max_len
   )
   # Tensors used to trace the model graph during conversion.
-  prefill_tokens = torch.full((1, prefill_seq_len), 0, dtype=torch.long)
+  prefill_tokens = torch.full((1, prefill_seq_len), 0, dtype=torch.int)
   prefill_input_pos = torch.arange(0, prefill_seq_len)
-  decode_token = torch.tensor([[0]], dtype=torch.long)
+  decode_token = torch.tensor([[0]], dtype=torch.int)
   decode_input_pos = torch.tensor([0], dtype=torch.int64)
```

Comment:
do you need to change here as well?

Reply:
Not immediately necessary since it's not going into the embedding.